bindings/tests/features/binding.feature

```diff
@@ -32,3 +32,7 @@ Feature: OpenDAL Binding
         Then The async file "test" entry mode must be file
         Then The async file "test" content length must be 13
         Then The async file "test" must have content "Hello, World!"
+
+    Scenario: OpenDAL Operations with Presign
+        Given A new OpenDAL Async Operator (s3 presign only)
```

Comment:
This test violates our VISION. OpenDAL should not write code directly against the service. The API call should either return a presigned request or `Unsupported` error.

Reply:
We need an available operator to test presign function here.
Maybe rename it to `Given A new OpenDAL Async Operator (presign available)`

Reply:
Or we can implement a fake presign function for `memory`.

Reply:
> We need an available operator to test presign function here.

No. Please don't mix bindings test with rust core's behavior test.

Binding test is used to make sure the binding implement correctly: the call is passed to rust core and got a response back from rust core. The presign itself doesn't need to be tested: they are covered by rust core's behavior test.

> Or we can implement a fake presign function for `memory`.

No. We can't implement this for `memory`.

Reply:
> But all of them sure be exact the same test suite.

So we can only write tests for `presign` after exposing `op.info()` and `Capability`. Is it right?

Reply:
> So we can only write tests for `presign` after exposing `op.info()` and `Capability`. Is it right?

No, we don't need to. We can simply call `presign` and check if its result is successful or returns an error with the error kind `Unsupported`.